common-components/index.md

```diff
@@ -421,44 +395,7 @@ Dynamics 365 provides various CRM features, such as:
 - Account Management
 - Enquiry Handling
 
-It can also be customised to support operational delivery, such as programme and financial assurance. Specific modules are also available for specific purposes, such as marketing and financial management.
-
-Dynamics 365 Portals provides an out-of-the-box front-end application that integrates directly with Dynamics 365. It can also be customised using CSS and Javascript, so can have a look and feel that is compliant with GOV.UK standards. For more complex implementations, it can also be integrated with a C#/.Net or Ruby application, to give more flexibility.
-
-Where an external-facing application is needed to support the CRM, Dynamics 365 Portals or development of a standalone web application can be used. Dynamics CRM can also be integrated with other external systems, such as support desks.
-
-__Thinking of implementing Anti-Virus and file scanning within Dynamics 365?__
```

Comment:
Have only just added this section in, to give visibility of the Cloudmersive implementation in Capital, which may be useful to people that do have Dynamics. Fine to tone down the 'sell' a bit imo, make it more factual - @Mazza76 to comment.

Reply:
Pretty much anything can be done, but architecture guidance should guide towards best practice strategic direction IMO. Putting web applications in front of CRM's is bad practice in my opinion, as it encourages the use of CRM's as acting as the relational database of an application. Which they are simply not a good solution for (again - its possible) but we have multiple examples (DQT, DTTP, TRAMs where this has caused many issues). The way we use CRMs as low code "quick fixes" is bad practice - CRM external portals are a prime example of bending CRMs into perceived quick ways to get a digital service up and running. Again we have many examples where this quickly becomes technical debt and not fit for purpose.

Reply:
Not sure this is the platform for this type of discussion? Let us have a catch-up to discuss further